JS/imdb.js

```diff
@@ -0,0 +1,62 @@
+
+function handle(e){
+  
+  //tirgers the "enter" event
+  if(e.keyCode === 13){
+  	
+  	// Ensure it is only this code that run
+    e.preventDefault(); 
+    var x = document.getElementById("search");
+    console.log(x.value);
+   }
+  
+ };
+
+function addMovie (e) {
+
+  // assign to x,y,z the relevant text element
+  var x = document.getElementById("movieName");
+  var y = document.getElementById("movieJanner");
+  var z = document.getElementById("movieTime");
+ 
+  //create new "li" item in order to insert it to the existing "ul"
+  var newItem = document.createElement("li");
```

Comment:
@oregilron One of the differences between code that work to a quality code and between developer to a good developer is conciseness.
concise definition: 

> expressing or covering much in few words;

This should be a rule of thumb, express much in few words.
the logic you are trying to achieve in lines 23-43 can be achieved with fewer lines of code and operations. try to improve it.